public/stylesheets/site/2.0/17-zone-home.css

```diff
@@ -74,12 +74,18 @@ user home, collections and challenges home, tag home, admin comms home
 /*(with filters)*/
 
 /* wrap long text in Firefox (applied to inbox only to avoid weirdness on all forms in IE) */
-#inbox-form {
-  float: left;
+form.inbox {
   overflow-x: auto;
   width: 75%;
 }
 
+form.inbox fieldset {
```

Comment:
The fact that the inbox had some seriously excessive space on its sides in all the layouts became crystal clear upon adjusting the page margins.